### DISCUSSION TITLE
Another effort to support J2ME apps

### DISCUSSION THREAD

**Discussion by jan-kleks** [SquirrelJME/SquirrelJME#1] [General Questions]
[It looks like they focus on games.](https://retropie.org.uk/forum/topic/11441/would-you-like-to-play-nokia-j2me-games-on-retropie/) I don't know, maybe you could cooperate in some way?

BTW, check out this (I have already sent you a message through reddit, but I don't know if you got it): [Anyone interested to start a bounty for a j2me like core for retroarch?](https://www.reddit.com/r/emulation/comments/70rfye/anyone_interested_to_start_a_bounty_for_a_j2me/)

Regards

**Comment by XerTheSquirrel** [comment]
Hello,

For cooperation, I do not have any interest in cooperating due to major differences in project goals along with licensing considerations. For that and the bounty, I do have another job and I am unable to commit myself full time to working on any project because of this. For any time I do spend programming, I generally prefer to stick to a single project at a time. I have replied to the comment in Reddit: <https://www.reddit.com/r/emulation/comments/70rfye/anyone_interested_to_start_a_bounty_for_a_j2me/dn8k8xq/>.

Closing this issue because it is really out of scope for this project. Really any such inquiries should be e-mailed to: Stephanie Gawroriski <xer@multiphasicapps.net>.

Thanks.
